### DISCUSSION TITLE
[Ultima VII] Crash to desktop

### DISCUSSION THREAD

**Discussion by sanviler** [exult/exult#548] [General]
Hello, since sometime (I am kinda at half game) game has become unplayable since it crashes back to desktop everytime my party engages a combat. Activating combat mode is ok, it doesn't trigger the crash, but as soon as someone tries to hit someone else, the game crashes back to desktop. I don't know why it started to do this, since it has been always playable. It could be determined on what gear (specially weapons) my party is using?

**Comment by GrantWChapman** [comment]
What game are you having the problem in? Black Gate with or without Forge of Virtue? Serpent Isle with or without Silver Seed? Serpent Isle Beta? One of the mods like BG Keyring?
If you have your save available I'll take a look and see if I can find anything.

**Comment by wench** [comment]
This is a known problem when playing with the 1.9 release and probably older. This problem has been fixed in snapshots or if you build exult from git yourself. We hope to release 1.10 with this fixed soon

**Comment by sanviler** [comment]
@GrantWChapman Black Gate with Forge of Virtue and BG Keyring.

After some experiments of various combination of party members and gear, I have found out that the issue was Jaana using a ligthing wand (which probably ran out of charges?). Once I have equipped her with a different weapon (now she uses a very sexy magical whip) the issue had gone.

The error I used to receive in the sterr was something to do with "weak_ptr", but at the moment I can't recall it, since I'm on a different system and Exult is on a PC  that I will use again on monday :P

If you need further infos, just let me know!

P.S.: do wands run out of charges? Because I tried to equip her with a blue and a red one (wind and fire I guess), but when combat begins, she switches weapon.

Thanks!!

**Reply by NotStanley4330** [reply]
Yes the wands do run out of charges
https://wiki.ultimacodex.com/wiki/Fire_Wand

**Reply by GrantWChapman** [reply]
Some wands do run out of charge. Lightning and fire wands for example, while magician's wands don't. If I remember correctly the DOS versions would delete the wand when charges run out but I seem to remember newer versions of Exult don't. That might have been changed to give the enchant item spell more use.
